### PR TITLE
[4.0] no jquery workflow button

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
- Removes a `jQuery('something').nextAll().toggle('else');`

FWIW https://plainjs.com got all the answers you need.
Eg for this one:
https://plainjs.com/javascript/traversing/get-siblings-of-an-element-40/ 

### Testing Instructions
Workflow in the articles still works fine

### CODE REVIEW

**Comment by bembelimen** [comment]
I got the following error:
grafik

**Comment by dgrammatiko** [comment]
@bembelimen that's because I'm an idiot, https://github.com/joomla/joomla-cms/pull/23417/files#diff-4bf6113582ca75b8b0e2f8e2dd64d87aR49
this needs to be `"Joomla.toggleAllNextElements(this, 'd-none')"`

**Comment by bembelimen** [comment]
grafik

I sent you a PR to fix this error.

File: build/media_src/com_content/js/admin-articles-workflow-buttons.es6.js [hunk 1]
```diff
@@ -3,6 +3,37 @@
...
+Joomla = window.Joomla || {};
+
+/**
+ * Method that switches a given class to the following elements of the element provided
+ *
+ * @param {HTMLElement}  element    The reference element
+ * @param {string}       className  The class name to be toggled
+ */
+Joomla.toggleAllNextElements = (element, className) => {
+  const siblings = [];
```
Comment (bembelimen) [inline build/media_src/com_content/js/admin-articles-workflow-buttons.es6.js]: Is there a reason to have this declaration outside of the getNextSiblings function?

Comment (dgrammatiko) [reply build/media_src/com_content/js/admin-articles-workflow-buttons.es6.js]: nope

**Comment by dgrammatiko** [comment]
@bembelimen should be fine now

BTW as these buttons are part of the JGrid (or whatever is the name of the class that builds the table rows) I would suggest to make the layout more generic (in the root/layouts) and also move the js file to somewhere like media/system/layouts/whatever-name-of-the-layout/name-of-the-script.

**Comment by bembelimen** [comment]
I have tested this item :white_check_mark: successfully on e4268100c499e4889c7c1240dbdfebc773e34c72<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23417">issues.joomla.org/tracker/joomla-cms/23417</a>.</sub>

**Comment by fancyFranci** [comment]
I have tested this item :white_check_mark: successfully on e4268100c499e4889c7c1240dbdfebc773e34c72<br /><br />No error messages and workflow is working correctly.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23417">issues.joomla.org/tracker/joomla-cms/23417</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23417">issues.joomla.org/tracker/joomla-cms/23417</a>.</sub>

**Comment by dgrammatiko** [comment]
@wilsonge

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**